src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ProvideCorrectArgumentsToFormattingMethodsTests.cs

```diff
@@ -15,10 +15,10 @@ namespace Microsoft.NetCore.Analyzers.Runtime.UnitTests
 {
     public class ProvideCorrectArgumentsToFormattingMethodsTests
```

Comment:
Can we add a test for `__arglist`?

Reply:
Do you mean an empty arglist? We have some tests with `__arglist(5)`

Reply:
yep

---
In reply to: [499685668](https://github.com/dotnet/roslyn-analyzers/pull/4226#discussion_r499685668) [](ancestors = 499685668)

Reply:
I am not sure how to get it to work, I was expecting the following to be good but that's not the case:

```csharp
public void M(__arglist)
{
    System.Console.WriteLine("", __arglist);
}
```